README.md

```diff
@@ -1 +1,79 @@
-# configuration-caas
+# Cluster-as-a-Service base configuration
+
+This repo provides a configuration for a Cluster-as-a-Service, built on top of
+Crossplane. This configuration exposes a simple API to your internal developers
+for creating Kubernetes clusters. In the background, this configuration can create clusters in AWS, Azure, and GCP. It automatically applies common recommended practices, such as connecting the cluster to existing Flux deployments, automating VPC setup, and creating a Flux operator in a cluster.
+
+This repo is a starting point for you to deliver your own
+Cluster-as-a-Service. Fork this repository and customize the configuration to
+meet your teams' needs.
+
+To deploy a multicloud CaaS control plane in your organization, check out the
```

Comment:
"CaaS" may be obvious to us, but I wonder if you want to be explicit above, like:

"This repo provides a configuration for a Cluster-as-a-Service (CaaS) ..."

Reply:
Thanks Alain! I've updated the acronym throughout the doc.

Reply:
I'm probably not qualified to approve this PR, but it looks good to me. Thanks!